public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,387 @@
+
+  `FormGroup` is a class, so you can create an instance and pass in child controls. In case you hadn’t noticed, 
+  it’s already in the component class. `FormBuilder` uses it in `this.form = this._fb.group({})`. 
+  The `.group()` method uses the `FormGroup` class to make a new group using the arguments you pass in.
+
+  In the form right now, the arguments passed into the `.group()` method are the `FormControl`s such as 
+  `name: '', username: ''`,  etc. It's straightforward but this form needs more than just one big group 
+  to keep things clear. 
+
+  To make this form more manageable, you can group the address **FormControls**. Start by 
+  adding `address: this._fb.group({ })` 
+  to the constructor and moving the address related FormControls inside it like so:
+
++makeExample('reactive-forms/ts/app/hero-signup-versions.component.ts', 'v4','app/hero-signup.component.ts (excerpt)')(format=".")
+
+:marked
+  You’ve changed the structure of the form model, so head on over to the template, give the new section a title of 
+  _Fortress of Solitude_ and wrap the address related **FormControls** in a `div` using the `formGroupName` directive. 
+  Pass it the name you gave it in the component class, `address`. 
+  This registers a `FormGroup` instance on the `Form` instance.
+
++makeExample('reactive-forms/ts/app/hero-signup-4.component.html', 'add-group','app/hero-signup.component.html (excerpt)')(format=".")
+
+:marked
+  Now the JSON output in the browser shows that `FormBuilder` has created a `FormGroup` for `address` and it is 
+  returned as an object:
+
+figure.image-display
+  img(src="/resources/images/devguide/reactive-forms/address-group.png" width="400px" alt="JSON output")
+
+:marked
+  Great! You’ve made a group and you can see that the template and the model are talking to one another by 
+  inspecting the form’s value, but you don’t always need the whole form at once. Sometimes you need to see what’s 
+  happening on one `FormControl`. Time for an aside.
+
+.l-main-section
+a#properties
+:marked
+  ## Inspecting Form Properties
+  A `FormControl` is just a class and if you create an instance, you can inspect its properties. 
+  That means that you can check to see exactly what’s happening with a given `FormControl`. 
+  For example, say you instantiate a new `formControl` (you don’t need to put this in your app):
+
+code-example(format=".").
+  const myControl = new FormControl();
+
+:marked
+  You can easily look at its properties:
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="10%")
+  col(width="90%")
+  tr
+    th Property
+    th Description
+  tr
+    td <code>myControl.value</code>
+    td
+      :marked
+         Returns the value of a `FormControl`.
+  tr
+    td <code>myControl.status</code>
+    td
+      :marked
+         Returns the status of a `FormControl` as valid, invalid - **KARA, CAN YOU HELP ME HERE?**.
```

Comment:
See this link: https://angular.io/docs/ts/latest/api/forms/index/AbstractControl-class.html#!#invalid-anchor
And if you scroll a bit up and down, it covers the other status values as well. Feel free to DM me if you have any questions about these.